frontend/pages/events.js

```diff
@@ -0,0 +1,361 @@
+import React, { useEffect, useState } from "react";
+import Link from "next/link";
+import Image from "next/image";
+import styles from "../styles/events.module.css";
+import PageHeader from "../components/header";
```

Comment:
Fix the following error with the header component 

> warn  - ./components/Header.js
> There are multiple modules with names that only differ in casing.
> This can lead to unexpected behavior when compiling on a filesystem with other case-semantic.

Reply:
Donee !